templates/inspec/examples/google_appengine_standard_app_version/google_appengine_standard_app_version.erb

```diff
@@ -0,0 +1,14 @@
+<% gcp_project_id = "#{external_attribute('gcp_project_id', doc_generation)}" -%>
+<% gcp_location = "#{external_attribute('gcp_location', doc_generation)}" -%>
+<% standardappversion = grab_attributes['standardappversion'] -%>
+
+describe google_appengine_standard_app_version(project: <%= gcp_project_id -%>, location: <%= gcp_location -%>, version_id: <%= doc_generation ? "'#{standardappversion['version_id']}'" : "standardappversion['version_id']" -%>, service: <%= doc_generation ? "'#{standardappversion['service']}'" : "standardappversion['service']" -%>) do
+  it { should exist }
+  its('service') { should eq <%= doc_generation ? "'#{standardappversion['service']}'" : "standardappversion['service']" -%> }
+  its('name') { should eq <%= doc_generation ? "'#{standardappversion['name']}'" : "standardappversion['name']" -%> }
+  its('id') { should eq <%= doc_generation ? "'#{standardappversion['version_id']}'" : "standardappversion['version_id']" -%> }
+  its('env_variables') { should eq <%= doc_generation ? "'#{standardappversion['env_var_value']}'" : "standardappversion['env_var_value']" -%> }
+  its('runtime_api_version') { should eq <%= doc_generation ? "'#{standardappversion['runtime_api_version']}'" : "standardappversion['runtime_api_version']" -%> }
+  its('runtime') { should eq <%= doc_generation ? "'#{standardappversion['runtime']}'" : "standardappversion['runtime']" -%> }
+  its('entrypoint') { should eq <%= doc_generation ? "'#{standardappversion['entrypoint']}'" : "standardappversion['entrypoint']" -%> }
```

Comment:
Upon testing it seems like `entrypoint` isn't actually returned by the API. It would probably make sense to mark it as `exclude` in the inspec.yaml in this case, as we can never retrieve it

Reply:
I think several of the other fields in this resource are also not returned by the API. Not sure why, but I'm not seeing environment variables coming back, or a couple other things. You can test this out with the sidebar at https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps.services.versions/get

Reply:
I have tested this locally. Many are not coming variable as it was not mentioned in this terraform. I have tested it locally. I have changed the environment, entry point and source variable. Cloud you please help and check again.